website/docs/r/incident_workflow_trigger.html.markdown

```diff
@@ -0,0 +1,86 @@
+---
+layout: "pagerduty"
+page_title: "PagerDuty: pagerduty_incident_workflow_trigger"
+sidebar_current: "docs-pagerduty-resource-incident-workflow-trigger"
+description: |-
+  Creates and manages an incident workflow trigger in PagerDuty.
+---
+
+# pagerduty\_incident\_workflow\_trigger
+
+An Incident Workflow Trigger defines when and if an Incident Workflow will be triggered.
```

Comment:
For being consistent with the rest of the resources' documentation, please use the **Incident Workflow Trigger** resource name for referencing the documentation of the product. e.g.: `[Incident Workflow Trigger](https://support.pagerduty.com/docs/<product-docs-page-here>`

Reply:
Added